packages/core/src/queries/sign-in-experience.test.ts

```diff
@@ -29,43 +28,47 @@ describe('sign-in-experience query', () => {
   };
 
   it('findDefaultSignInExperience', async () => {
-    const expectSql = sql`
-      select ${sql.join(Object.values(fields), sql`, `)}
-      from ${table}
+    /* eslint-disable sql/no-unsafe-query */
+    const expectSql = `
+      select "id", "branding", "language_info", "terms_of_use", "forget_password_enabled", "sign_in_methods", "social_sign_in_connector_ids"
+      from "sign_in_experiences"
+      where "id" = $1
     `;
+    /* eslint-enable sql/no-unsafe-query */
```

Comment:
nit: would it work to wrap up in the slonik SQL? Wondering why we have to make the change here

Reply:
or lint will fail
![image](https://user-images.githubusercontent.com/10594507/159448375-78d9ebb3-26bc-4eed-a162-5eaa895bb2cb.png)